test/Microsoft.AspNet.Mvc.Razor.Host.Test/MvcRazorHostTest.cs

```diff
@@ -47,6 +47,39 @@ public class MvcRazorHostTest
             Assert.Equal("src/file.cshtml", chunkInheritanceUtility.InheritedCodeTreePagePath, StringComparer.Ordinal);
         }
 
+        [Theory]
+        [InlineData("//")]
+        [InlineData("C:/")]
+        [InlineData(@"\\")]
+        [InlineData(@"C:\")]
+        public void DecorateCodeBuilder_DesignTimeRazorPathNormalizer_NormalizesChunkInheritanceUtilityPaths(
+            string rootPrefix)
+        {
+            // Arrange
+            var rootedAppPath = $"{rootPrefix}SomeComputer/Location/Project/";
```

Comment:
Could you add a test where the `rootedFilePath` and `rootedAppPath` don't have the same line ending formats

For instance: `filePath`: "/foo/bar/baz.cshtml", `appPath`: "foo\bar\baz.cshtml"